crystal/src/y3storage/autoinst_issues/missing_root.cr

```diff
@@ -0,0 +1,49 @@
+# Copyright (c) [2017] SUSE LLC
+#
+# All Rights Reserved.
+#
+# This program is free software; you can redistribute it and/or modify it
+# under the terms of version 2 of the GNU General Public License as published
+# by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+# FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License for
+# more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, contact SUSE LLC.
+#
+# To contact SUSE LLC about this file by physical or electronic mail, you may
+# find current contact information at www.suse.com.
+
+require "./issue"
+
+module Y3Storage
+  module AutoinstIssues
+    # The proposal was successful but there is not root partition (/) defined.
+    #
+    # This is a fatal error because the installation is not possible.
+    class MissingRoot < Issue
+      def initialize(*args)
+        super
```

Comment:
is it needed at all?

Reply:
I just copied this directly from the Ruby code.

Very likely it was in the Ruby code because it used to have a call to `textdomain`.  Now is useless in both the Ruby and the Crystal code.